DEV-9/StringSubstitute.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+
+namespace RandomChangeTheSymbolsInString
+{
+  //This class consists of the methods which let us to set the random positions and the random length of the lines's part
+  //and allowes to replace this parts in established positions
+  class StringSubstitute
+  {
+    const int AMOUNTOFLINES = 2;
+
+    public string SourceLine { get; set; }
+    public string LineForSubstitution { get; set; }
+    public Random rnd = new Random();
+
+    //Field initialization:
+    public StringSubstitute(string sourceLine, string lineForSubstitution)
+    {
+      SourceLine = sourceLine;
+      LineForSubstitution = lineForSubstitution;
+    }
+
+    //This method lets us replace this part of the line, with random length from a random positions,
+    //in the source line, to random positions in another one
+    public string Substitute()
+    {
+      int[] indexOfStart = RandomPosition(SourceLine.Length, LineForSubstitution.Length);
+      int[] length = RandomLength(SourceLine.Length, LineForSubstitution.Length, indexOfStart);
+      string substring = SourceLine.Substring(indexOfStart[0], length[0]);
+      string resultAfterSubstitution = LineForSubstitution.Remove(indexOfStart[1], length[1]);
+      resultAfterSubstitution = resultAfterSubstitution.Insert(indexOfStart[1], substring);
+      return resultAfterSubstitution;
+    }
+
+    //For setting the random position in which we cut out the part of the line
+    //to the random position where we insert this part
+    public int[] RandomPosition(int lengthOfSourceLine, int lengthOfLineForSubstitute)
```

Comment:
Very bad idea to form two start position for two separate string in one method. in this case the string are too linked each to other